frontend/apps/reader/modules/readermenu.lua

```diff
@@ -79,7 +79,9 @@ function ReaderMenu:init()
 end
 
 function ReaderMenu:getPreviousFile()
-    return require("readhistory"):getPreviousFile(self.ui.document.file)
+    if not self.ui.document then
+        return require("readhistory"):getPreviousFile(self.ui.document.file)
+    end
```

Comment:
Or maybe just this?
```suggestion
```suggestion
return require("readhistory"):getPreviousFile((self.ui.document or {}).file)
```
```

Reply:
Please no :) Even I had difficulties understanding this ugly construct :) Better something clear and explicite.

Reply:
Suddenly I can't reproduce the crash with the unmodified sources anymore. So I can not tell if this solves the problem.
I will look into it.

Reply:
I am a step further: (got some backups from my NAS). 
The crash is reproducable.

My problem was, that I had two gestures at the same swipe: `previous document` and `changesize`. 

My proposed solution for the `getPreviosFile` masked the second "changesize" event somehow.
So I can reproduce it with only one event in gesture -> The crash log is also suspicious:
```
05/15/21-19:53:34 DEBUG refresh on physical rectangle 165 5 210 53
./luajit: frontend/apps/reader/modules/readerfont.lua:214: attempt to index field 'document' (a nil value)
stack traceback:
	frontend/apps/reader/modules/readerfont.lua:214: in function 'handleEvent'
	frontend/ui/widget/container/widgetcontainer.lua:95: in function 'propagateEvent'
```
-> It is in readerfont.lua:214

`lastfile` has nothing to do it!

Reply:
In which context, though? That gesture should only be available in CRe, I think, so i'm not quite sure how `document` could be unset while, well, inside a document ;p.

Reply:
> Better something clear and explicite.

I find it clear and explicit, but alright. :-P